gc/base/standard/ConfigurationGenerational.hpp

```diff
@@ -64,8 +74,18 @@ class MM_ConfigurationGenerational : public MM_ConfigurationStandard
 	bool initialize(MM_EnvironmentBase* env);
 	MM_MemorySubSpaceSemiSpace *createSemiSpace(MM_EnvironmentBase *envBase, MM_Heap *heap, MM_Scavenger *scavenger, MM_InitializationParameters *parameters, UDATA numaNode = UDATA_MAX);
 	virtual void tearDown(MM_EnvironmentBase* env);
+	/**
+	 * Sets the number of gc threads
+	 *
+	 * @param env[in] - the current environment
+	 */
+	virtual void initializeGCThreadCount(MM_EnvironmentBase* env);
 private:
 	uintptr_t calculateDefaultRegionSize(MM_EnvironmentBase *env);
+
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+	void initializeConcurrentScavengerThreadCount(MM_EnvironmentBase* env);
+#endif
```

Comment:
missing endif comment: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif
```suggestion
#endif /* defined(OMR_GC_CONCURRENT_SCAVENGER) */
```